src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx

```diff
@@ -61,53 +84,49 @@ class ConfirmationPage extends React.Component {
     return (
       <div>
         <h3 className="confirmation-page-title">Your claim has been submitted.</h3>
-        <p>We usually process claims within <strong>99 days</strong>.</p>
-        <p>We may contact you if we have questions or need more information.
-        You can print this page for your records.</p>
-        { selected4142 && privateRecordReleaseContent}
-        <h4 className="confirmation-guidance-heading">
-          What happens after I apply?
-        </h4>
-        <p className="confirmation-guidance-message">
-          You don’t need to do anything unless we send you a letter asking for
-          more information. You can check the status of your claim online. The
-          time frame you see may vary based on how complex your claim is.
+        <p>
+          We process applications in the order we receive them.
+          We may contact you if we have questions or need more information.
+          You can print this page for your records.
         </p>
-        <a href="/disability-benefits/track-claims">
-          Check the status of your disability claim.
-        </a>
-        <br/>
-        <a href="/disability-benefits/after-you-apply/">
-          Learn more about what happens after you file a disability claim.
-        </a>
+        { selected4142 && privateRecordReleaseContent}
+
         <div className="inset">
-          <h4>Disability Compensation Claim for Increase <span className="additional">(Form 21-526EZ)</span></h4>
+          <h4>Disability Compensation Claim <span className="additional">(Form 21-526EZ)</span></h4>
           <span>
             For {first} {middle} {last} {suffix}
           </span>
           <ul className="claim-list">
-            <strong>Conditions claimed</strong>
+            <strong>Conditions claimed for increase</strong>
             <br/>
-            <ul className="disability-list">
-              {disabilities.filter(item => item['view:selected']).map((disability, i) => {
+            {/* <ul className="disability-list">
```

Comment:
What's the plan for this code?

Reply:
Y'know, that's a great question. Kinda looks like I commented this out, doesn't it? I was just asking Mark yesterday why it was commented and he said it wasn't in master. :stuck_out_tongue:

I think I commented it out so I could actually get to the disabilities screen for some reason during testing and I just never cleaned it up. Guess that means I can go ahead and uncomment it! :grimacing: